### PR TITLE
README Updates

### PR DESCRIPTION
**Pull request by carlsverre**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,9 +32,38 @@ MemSQL Spark Connector
 The MemSQL Spark connector provides tools for reading from and writing to
 MemSQL databases in Spark.
 
-There are two parts to the connector; the MemSQLRDD and the saveToMemsql
-function.  The former allows users to read data out of a MemSQL database; the
-latter allows users to write to a MemSQL database.
+The connector provides a number of intergrations with Apache Spark including a custom RDD type, dataframe helpers and a MemSQL Context.
```
Comment (wsong) [inline README.md]: Typo on integrations.

Comment (wsong) [inline README.md]: Also, dataframe is properly capitalized as "DataFrame".

File: README.md [hunk 2]
```diff
@@ -32,9 +32,38 @@ MemSQL Spark Connector
...
+
+###MemSQLContext
+
+The MemSQL Context maintains metadata about a MemSQL cluster and extends the Spark SQLContext.
+
+```
+import com.memsql.spark.connector.context.MemSQLContext
```
Comment (choochootrain) [inline README.md]: this should be `com.memsql.spark.context.MemSQLContext`

**Comment by choochootrain** [comment]
lgtm

**Pull request merged**